tools/protoxform/protoprint.py

```diff
@@ -580,6 +622,27 @@ def visit_service(self, service_proto, type_context):
     def visit_enum(self, enum_proto, type_context):
         if protoxform_options.has_hide_option(enum_proto.options):
             return ''
+        # Verify that not hidden deprecated enum values of non-frozen protos have valid version
+        # annotations.
+        for v in enum_proto.value:
+            if v.options.deprecated and not self._frozen_proto and \
+                    not protoxform_options.has_hide_option(v.options):
+                # If the enum value has annotation from deprecation.proto, need to import it.
+                if v.options.HasExtension(deprecation_pb2.deprecated_at_minor_version_enum) or \
+                        v.options.HasExtension(deprecation_pb2.disallowed_by_default_enum):
+                    self._needs_deprecation_annotation_import = True
+                if v.name != 'DEPRECATED_AND_UNAVAILABLE_DO_NOT_USE':
```

Comment:
We should have a named constant for that.

Reply:
Done.